packages/ui-rating/src/Rating/index.tsx

```diff
@@ -50,8 +51,8 @@ class Rating extends Component<RatingProps> {
 
   static defaultProps = {
     animateFill: false,
-    // @ts-expect-error ts-migrate(7006) FIXME: Parameter 'filled' implicitly has an 'any' type.
-    formatValueText: (filled, iconCount) => `${filled} / ${iconCount}`,
+    formatValueText: (filled: string, iconCount: string) =>
```

Comment:
Once you fix the types below, this will be: 

```suggestion
    formatValueText: (filled: number, iconCount: number) =>
```